src/ProgressOnderwijsUtils/SchemaReflection/DbColumnMetaData.cs

```diff
@@ -97,6 +97,13 @@ public static DbColumnMetaData Create(string name, Type dataType, bool isKey, in
 
         public SqlXType UserTypeId { get; set; }
         public short MaxLength { get; set; } = SchemaReflection.SqlTypeInfo.VARCHARMAX_MAXLENGTH_FOR_SQLSERVER;
+
+        public bool IsString
+            => UserTypeId == SqlXType.VarChar || UserTypeId == SqlXType.NVarChar || UserTypeId == SqlXType.NChar || UserTypeId == SqlXType.Char;

```

Comment:
We hebben al ergens een mapping van sql type naar C# (bijv. bij de generatedpocos in gebruik), kun je checken of die hier bruikbaar is?

Reply:
ik neem aan dat je hierop doelt? SqlXTypeExtensions.typeLookup
```
            (typeof(string), SqlXType.Char),
            (typeof(string), SqlXType.NChar),
            (typeof(string), SqlXType.VarChar),
            (typeof(string), SqlXType.Xml),
            (typeof(string), SqlXType.SysName),
            (typeof(char), SqlXType.NChar),
```
Ik moest wel even 2x kijken waarom de laatste regel niet fout zou gaan, maar door de volgorde gaat de mapping van NChar naar string wel goed.

Ik zal kijken of dit te gebruiken is. Lijkt me wel dat de C# char NIET onder de IsString moet vallen.

Reply:
Is dat ook indirect wat de GeneratedPocos gebruiken?  Ik zou iig verwachten dat er een of andere handige utility wrapper omheen zit.